src/Graphics/Sprite.cpp

```diff
@@ -223,28 +197,18 @@ void Sprite::render(int x, int y, bool transparency, bool light, int outline, un
 void Sprite::renderCropped(int x, int y, int dx, int dy, unsigned int width, unsigned int height, bool transparency,
                            bool light, unsigned int lightValue)
 {
-    std::vector<glm::vec2> vertices;
-    std::vector<glm::vec2> UV;
-
-    glm::vec2 vertex_up_left    = glm::vec2( (float)x, (float)y);
-    glm::vec2 vertex_up_right   = glm::vec2( (float)(x+width), (float)y);
-    glm::vec2 vertex_down_right = glm::vec2( (float)(x+width), (float)(y+height));
-    glm::vec2 vertex_down_left  = glm::vec2( (float)x, (float)(y+height));
-
-    vertices.push_back(vertex_up_left   );
-    vertices.push_back(vertex_down_left );
-    vertices.push_back(vertex_up_right  );
-    vertices.push_back(vertex_down_right);
-
-    glm::vec2 uv_up_left    = glm::vec2( (float)dx/(float)_texture->textureWidth(), (float)dy/(float)_texture->textureHeight() );
-    glm::vec2 uv_up_right   = glm::vec2( (float)(dx+width)/(float)_texture->textureWidth(), (float)dy/(float)_texture->textureHeight() );
-    glm::vec2 uv_down_right = glm::vec2( (float)(dx+width)/(float)_texture->textureWidth(), (float)(dy+height)/(float)_texture->textureHeight() );
-    glm::vec2 uv_down_left  = glm::vec2( (float)dx/(float)_texture->textureWidth(), (float)(dy+height)/(float)_texture->textureHeight() );
-
-    UV.push_back(uv_up_left   );
-    UV.push_back(uv_down_left );
-    UV.push_back(uv_up_right  );
-    UV.push_back(uv_down_right);
+	glm::vec2 vertices[4] = {
+		glm::vec2((float)x, (float)y),
+		glm::vec2((float)x, (float)(y + height)),
+		glm::vec2((float)(x + width), (float)y),
+		glm::vec2((float)(x + width), (float)(y + height))
+	};
+    glm::vec2 UV[4] = {
+		glm::vec2((float)dx / (float)_texture->textureWidth(), (float)dy / (float)_texture->textureHeight()),
+		glm::vec2((float)dx / (float)_texture->textureWidth(), (float)(dy + height) / (float)_texture->textureHeight()),
+		glm::vec2((float)(dx + width) / (float)_texture->textureWidth(), (float)dy / (float)_texture->textureHeight()),
+		glm::vec2((float)(dx + width) / (float)_texture->textureWidth(), (float)(dy + height) / (float)_texture->textureHeight())
+	};
```

Comment:
Please follow the style guide. We don't use tabs. I already mentioned this before.